app/src/androidTest/java/net/skyscanner/backpack/docs/DocsRegistry.kt

```diff
@@ -194,6 +200,28 @@ private fun setupWholeMonthCalendar() {
     }
 }
 
+private fun setupCalendar2(rule: AndroidComposeTestRule<*, *>) {
```

Comment:
Again, since we cannot select things programmatically in Compose from this point in code, we simulate user interaction